app/views/users/show.html.haml

```diff
@@ -11,7 +11,7 @@
   %b Active:
   = @user.active
 %p
-  %b Admin:
+  %b STSPV:
   = @user.admin
```

Comment:
While we're here, can we change this to not show `true` or `false` in a view?

Reply:
What do you mean?

Reply:
I can't find it how do I rails/ruby/haml

Reply:
I mean that `@user.admin` is the value `true` or `false`. Ideally this would say "Yes" or "No".